webpack/assets/javascripts/react_app/components/hosts/powerStatus/PowerStatus.stories.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PowerStatus from './index';
+import {
+  Story,
+  Code,
+  Text,
+  StoryWithCustomState,
+} from '../../../../../../stories/components';
```

Comment:
why not using @stories interface?

Reply:
those storybook helpers still lives in Foreman,
so I think currently it is out of the scope of this PR to move them to foreman-js.

Reply:
can you please open an issue for it?

Reply:
opened https://projects.theforeman.org/issues/29393